### ISSUE TITLE
Not able to update 

### ISSUE THREAD

**Issue by drewthon** [GCalToolkit/Sheets2GCal#20]
I'm not able to update my spreadsheet.  It tells me the following...

Unregistered users can not update/delete unlimited events.
gin up in 2 clicks at www.sheets2GCal.com with this Google Account to remove some limits for free!

thanks

**Comment by GCalToolkit** [comment]
Yes, to increase the number of daily edits to calendars in your account you need a free account on www.Sheets2GCal.com with the same email address that you use for Google Sheets.

If that's already the case then you probably have several Google accounts open in the same browser. 

There's a bug in Google's add-on system which causes the wrong Google account to be used so Sheets add-ons always default to the "Default" Google user.

To work around this you can either use a separate Chrome profile, an incognito window, or log out of other Google accounts so the one you want to use becomes the default.
